### DISCUSSION TITLE
New Source: coreBOS

### DISCUSSION THREAD

**Discussion by charity1475** [airbytehq/airbyte#33185] [New Connector Request]
## Tell us about the new integration you’d like to have
Which source and which destination? Which frequency?

Answ:	coreBOS connector for (source)coreBOS CRM integration.

## Describe the context around this new integration
Which team in your company wants this integration, what for? This helps us understand the use case.

Answ: this connector is destined for all users of coreBOS CRM platform to easily synch data across variour platforms from 
coreBOS

## Describe the alternative you are considering or using
What are you considering doing if you don’t have this integration through Airbyte?

Answ: coreBOS connector for airbyte is intented to enhance usabilility and making data movement wayfar simple.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200367912513076/1200368088034125) by [Unito](https://www.unito.io)

**Comment by sherifnada** [comment]
@charity1475 Thanks for creating this issue! 🎉 Some questions: 

* What scale of data do you anticipate pulling from coreBOS? 
* Is this blocking for your ability to use Airbyte? 
* If Airbyte couldn't provide this, what is the alternative? 
* What other sources/ destinations would you use in conjunction with coreBOS? 
* Would you be open/able to contributing the source connector to Airbyte? We are happy to offer any needed support whether it's calls, pairing on code, answering questions, or anything else

[a comment by an automated account is not shown]
